lib/prawn/svg/extension.rb

```diff
@@ -20,6 +20,14 @@ def svg(data, options = {}, &block)
         { warnings: svg.document.warnings, width: svg.document.sizing.output_width,
           height: svg.document.sizing.output_height }
       end
+
+      def fill_svg_gradient(**kwarguments)
+        Prawn::SVG::GradientRenderer.new(self, :fill, **kwarguments).draw
+      end
+
+      def stroke_svg_gradient(**kwarguments)
+        Prawn::SVG::GradientRenderer.new(self, :stroke, **kwarguments).draw
+      end
```

Comment:
Not sure that these svg-specific methods should sit in the global namespace, I can't imagine why a user would want to call them directly?

Would `Prawn::SVG::Renderer` be an appropriate place for the calls instead, following the `svg:render_sub_document` example?

Reply:
Funnily that is exactly how I had it while I was working on it, but I refactored them into the global methods because I wanted to prove to myself that I wasn't doing anything inside the renderer that couldn't easily be ported over to Prawn itself. 😄 

But since that is proved, I'm happy to move it back into `Prawn::SVG::Renderer` as you suggest. I have pushed that change now.

Reply:
Cool.  If it made its way to Prawn it would be a bit odd to have it named `svg` but it would be a very worthy addition otherwise I think!  It seems the Prawn project is moving again after a couple of years of not.